### PR TITLE
ANDROID: Update to 3.8.0

### PR DESCRIPTION
**Pull request by amytang0**

### CODE REVIEW

**Review by jhurray — APPROVED** [review]

File: android/build.gradle [hunk 1]
```diff
@@ -65,7 +65,7 @@ repositories {
 
 dependencies {
     implementation "com.facebook.react:react-native:+"
-    implementation "com.plaid.link:sdk-core:3.7.1"
```
Comment (Japottatweet311) [inline android/build.gradle:68]: iOS

File: android/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -4,7 +4,7 @@
   <application>
     <meta-data
       android:name="com.plaid.link.react_native"
-      android:value="7.3.0" />
```
Comment (Japottatweet311) [inline android/src/main/AndroidManifest.xml:7]: iOS

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-native-plaid-link-sdk",
-  "version": "7.3.0",
```
Comment (Japottatweet311) [inline package.json:3]: 4

File: android/src/main/AndroidManifest.xml (hunk 2 above)
Comment (Japottatweet311) [reply android/src/main/AndroidManifest.xml:7]: 3

**Review by Japottatweet311 — COMMENTED** [review]
Need to email for my approval

**Pull request merged**